test/dotnet.Tests/dotnet.Tests.csproj

```diff
@@ -23,47 +21,49 @@
       <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
     </Content>
   </ItemGroup>
-  
-  <Target Name="WriteExpectedVersion"
-          BeforeTargets="BeforeBuild"
-          DependsOnTargets="PrepareForBuild">
-
+  <Target Name="WriteExpectedVersion" BeforeTargets="BeforeBuild" DependsOnTargets="PrepareForBuild">
     <PropertyGroup>
       <ExpectedVersionFileInIntermediateFolder>$(IntermediateOutputPath)ExpectedSdkVersion.txt</ExpectedVersionFileInIntermediateFolder>
-      
       <VersionFileContent>$(SdkVersion)</VersionFileContent>
       <ExistingVersionFileContent Condition=" Exists('$(ExpectedVersionFileInIntermediateFolder)') ">
         $([System.IO.File]::ReadAllText($(ExpectedVersionFileInIntermediateFolder)))
       </ExistingVersionFileContent>
-
       <ShouldOverwriteVersionFile>false</ShouldOverwriteVersionFile>
-      <ShouldOverwriteVersionFile
-        Condition=" '$(ExistingVersionFileContent.Trim())' != '$(VersionFileContent.Trim())' ">true</ShouldOverwriteVersionFile>
+      <ShouldOverwriteVersionFile Condition=" '$(ExistingVersionFileContent.Trim())' != '$(VersionFileContent.Trim())' ">true</ShouldOverwriteVersionFile>
     </PropertyGroup>
-    
-    <WriteLinesToFile File="$(ExpectedVersionFileInIntermediateFolder)"
-                      Lines="$(VersionFileContent)"
-                      Condition=" '$(ShouldOverwriteVersionFile)' == 'true' "
-                      Overwrite="true" />
-
+    <WriteLinesToFile File="$(ExpectedVersionFileInIntermediateFolder)" Lines="$(VersionFileContent)" Condition=" '$(ShouldOverwriteVersionFile)' == 'true' " Overwrite="true" />
     <ItemGroup>
       <Content Include="$(ExpectedVersionFileInIntermediateFolder)" CopyToOutputDirectory="PreserveNewest" />
       <FileWrites Include="$(ExpectedVersionFileInIntermediateFolder)" />
     </ItemGroup>
   </Target>
-
+  <Target Name="WriteExpectedTargetFrameworkMoniker" BeforeTargets="BeforeBuild" DependsOnTargets="PrepareForBuild">
```

Comment:
Should this be moved into the test project as well?

Reply:
There is a very similar existing pattern above `WriteExpectedVersion`